chains/container-chains/runtime-templates/frontier/src/lib.rs

```diff
@@ -481,6 +482,7 @@ impl pallet_transaction_payment::Config for Runtime {
     type WeightToFee = WeightToFee;
     type LengthToFee = ConstantMultiplier<Balance, TransactionByteFee>;
     type FeeMultiplierUpdate = SlowAdjustingFeeUpdate<Self>;
+    type WeightInfo = ();
```

Comment:
should we add this pallet to benchmarks now?

Reply:
Not in this PR please